velox/exec/tests/HashPartitionFunctionTest.cpp

```diff
@@ -154,3 +154,17 @@ TEST_F(HashPartitionFunctionTest, spec) {
     ASSERT_EQ(hashSpec->toString(), copy->toString());
   }
 }
+
+TEST_F(HashPartitionFunctionTest, noKeyAndBitRange) {
+  auto vector = makeRowVector({makeFlatVector<int32_t>({1, 2, 3, 4, 5, 6})});
+  auto rowType = asRowType(vector->type());
+  const auto numRows{vector->size()};
+  HashPartitionFunction function(4, rowType, {}, {});
+
+  std::vector<uint32_t> partitions(numRows);
+  auto partitionNum = function.partition(*vector, partitions);
+  EXPECT_EQ(partitionNum.value(), 0u);
+  for (size_t i = 0; i < numRows; ++i) {
+    EXPECT_EQ(partitions[i], 0);
```

Comment:
This check is no longer correct and just 'happens' to work.

Reply:
Oh I forgot this, thanks for the reminder.